src/util.rs

```diff
@@ -44,6 +44,25 @@ pub fn history() -> web_sys::History {
     window().history().expect("Can't find history")
 }
 
+/// Convenience function to access the `web_sys::HtmlDocument`.
+pub fn html_document() -> web_sys::HtmlDocument {
+    wasm_bindgen::JsValue::from(document()).unchecked_into::<web_sys::HtmlDocument>()
+}
+
+/// Convenience function to get all cookies from the current `HtmlDocument`
+#[cfg(feature = "cookies")]
```

Comment:
What's the reason for feature flag? Is app too big with `cookie` dependency?

Reply:
> What's the reason for feature flag?

I just don't want to force all users to depend on `cookie`

> Is app too big with `cookie` dependency?

no.

Reply:
> I just don't want to force all users to depend on cookie

I don't see problem with it . Otherwise we would have to document it, it would be another obstacle for new Seed users and code would be a little bit harder to reason about. I suggest to remove that feature flag and revisit it later, when we will be resolving https://github.com/David-OConnor/seed/issues/248.
What do you think?

Reply:
:+1: